yellowbrick/classifier/prcurve.py

```diff
@@ -570,6 +615,8 @@ def precision_recall_curve(
         model,
         ax=ax,
         classes=classes,
+        colors=colors,
+        cmap=cmap,
```

Comment:
Just to make sure that our Quick Methods have the same behavior as our regular visualizers, I've updated the params here too.